terraform/modules/aws-cargohold-asset-storage/resources.security_groups.tf

```
@@ -0,0 +1,19 @@
+# a security group for access to the private traffic between kubernetes nodes. should be added to all kubernetes nodes.
+resource "aws_security_group" "talk_to_S3" {
+  name        = "talk_to_S3"
+  description = "hosts that are allowed to talk to S3."
+  vpc_id      = var.vpc_id
+
+  # S3
+  egress {
```

```suggestion

  egress {
```
IMHO redundant to the resource description.